locales/en.default.schema.json

```
@@ -870,7 +870,7 @@
             "label": "Inverse"
           },
           "label": "Color scheme",
-          "info": "Visible when text box displayed"
+          "info": "Used for text box"
```

```suggestion
          "info": "Used for text box."
```
Just missing the period for info text.